### PR TITLE
Image icon link cards

### PR DESCRIPTION
**Pull request by milosyuki**
This PR includes Image icon link card component with 3 variants: `image_icon_link_card`, `icon_link_card` and `dapp` and is being showcased in the storybook as well. This is the component made of different components displayed in the figma as "Home page cards". The only one's missing are Large card and Community card which will be covered in the next PR.

Also, home card icons are created and added to the storybook.

### CODE REVIEW

File: _data/pages/en/developers.yml
Comment (robwalsh21) [inline _data/pages/en/developers.yml:1]: @milosyuki I'm noticing some rendering issues with the  cards under courses in dark mode  
CleanShot 2023-04-14 at 15 22 47

File: _data/pages/en/home.yml
Comment (robwalsh21) [inline _data/pages/en/home.yml:1]: @milosyuki Another example of rendering issue with smaller card
CleanShot 2023-04-14 at 15 26 30

**Comment by robwalsh21** [comment]
> This PR includes Image icon link card component with 3 variants: `image_icon_link_card`, `icon_link_card` and `dapp` and is being showcased in the storybook as well. This is the component made of different components displayed in the figma as "Home page cards". The only one's missing are Large card and Community card which will be covered in the next PR.
> 
> Also, home card icons are created and added to the storybook.

hey @milosyuki   Noticing some issues on hover with cards on home and developers page 
CleanShot 2023-04-18 at 10 58 12@2x
CleanShot 2023-04-18 at 10 57 54@2x

File: src/components/Icons/CardIcons/DappsIcon/DappsIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { CardIconProps } from "../IconInterface";
+
+const DappsIcon: React.FC<CardIconProps> = ({ gradientColor1, gradientColor2, filter, borderRadius }) => {
+  const gradientIdPrefix = `gradient-${Math.random().toString(36).substr(2, 9)}`;
```
Comment (thecotne) [inline src/components/Icons/CardIcons/DappsIcon/DappsIcon.tsx]: use this https://react.dev/reference/react/useId

File: src/app/[locale]/tutorials/(components)/TutorialsPage.tsx [hunk 2]
```diff
@@ -302,7 +302,7 @@ type Tutorial = {
 
 function CustomHits() {
   const { hits, showMore, isLastPage } = useInfiniteHits<Tutorial>();
-
+console.log('hits ', hits)
```
Comment (thecotne) [inline src/app/[locale]/tutorials/(components)/TutorialsPage.tsx]: this should be removed right?

File: src/blocks/cards/IconLinkCard.tsx [hunk 3]
```diff
@@ -58,7 +58,7 @@ export const IconLinkCard = ({
             justify="flex-end"
             alignItems="flex-start"
           >
-            <Image src={icon} mb="20px" alt={title} />
+            {/* <Image src={icon} mb="20px" alt={title} /> */}
```
Comment (thecotne) [inline src/blocks/cards/IconLinkCard.tsx]: should this be removed instead of commenting it?

**Review by robwalsh21 — APPROVED** [review]

File: src/components/Card/style.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
+/*! @algolia/autocomplete-theme-classic 1.7.4 | MIT License | © Algolia, Inc. and contributors | https://github.com/algolia/autocomplete */
```
Comment (thecotne) [inline src/components/Card/style.css]: this comment should be removed

**Review by thecotne — CHANGES_REQUESTED** [review]
looks good overall just few comments

File: src/components/Icons/CardIcons/DappsIcon/DappsIcon.tsx (hunk 1 above)
Comment (milosyuki) [reply src/components/Icons/CardIcons/DappsIcon/DappsIcon.tsx]: Thanks, fixed!

File: src/app/[locale]/tutorials/(components)/TutorialsPage.tsx (hunk 2 above)
Comment (milosyuki) [reply src/app/[locale]/tutorials/(components)/TutorialsPage.tsx]: Thanks, removed.

File: src/blocks/cards/IconLinkCard.tsx (hunk 3 above)
Comment (milosyuki) [reply src/blocks/cards/IconLinkCard.tsx]: Thanks, removed.

File: src/components/Card/style.css (hunk 4 above)
Comment (milosyuki) [reply src/components/Card/style.css]: Thanks, removed.

**Review by thecotne — APPROVED** [review]
👍

File: _data/pages/en/developers.yml
Comment (milosyuki) [reply _data/pages/en/developers.yml:1]: Thanks, fixed.

File: _data/pages/en/home.yml
Comment (milosyuki) [reply _data/pages/en/home.yml:1]: Thanks, fixed.

**Comment by robwalsh21** [comment]
Did you get a chance to look at this @cxramina  ?

**Comment by cxramina** [comment]
> Did you get a chance to look at this @cxramina ?

Yes, I send comments to Milos in Slack

**Comment by robwalsh21** [comment]
Great.  Thanks

**Pull request merged**